### DISCUSSION TITLE
The use of both the playwright and the @playwright/test packages

### DISCUSSION THREAD

**Discussion by cc359** [akshayp7/playwright-typescript-playwright-test#32] [Ideas]
Hi,

Correct me if I'm wrong but as far as I understand it, only the `@playwright/test` package is needed for this project. As it includes everything, and more, contained within the `playwright`package. 

Thanks.

**Comment by akshayp7** [comment]
Hi Chris thanks for your feedback let me check into this and let you know.

**Comment by akshayp7** [comment]
In playwright.config.ts, the configuration we use for devices (emulation), comes from playwright package, so I have kept both of them

**Reply by cc359** [reply]
Hi Akshay, 

`devices` is available in `@playwright/test`, as is demonstrated [in the docs](https://playwright.dev/docs/test-configuration#emulation ). 

Hope all is well :)

**Reply by akshayp7** [reply]
Yes thanks let me check it out and update my repo accordingly. Thank you so much for bringing this into my notice.

**Comment by akshayp7** [comment]
Hi Chris actually you are right for some reason previously the devices were not getting detected from playwright/test package, but now I checked they are working correctly. Thank you so much for your feedback only playwright/test package is required I will make changes in my next release.

**Comment by akshayp7** [comment]
Hey Chris your suggestion is included in v0.18.0. Thanks for your feedback
